src/component/Maps/maps.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./maps.scss";
+import GoogleMapReact from "google-map-react";
+
+import { Icon } from "@iconify/react";
+import locationIcon from "@iconify/icons-mdi/map-marker";
+
+const google_maps_key = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
```

Comment:
Please add this env variable to the `.env.sample` file so we know what variables we need locally when we develop. set it equal to `example` or something like that. don't push up the actual token